superset/assets/javascripts/explorev2/stores/fields.js

```diff
@@ -1140,6 +1140,20 @@ export const fields = {
     mapStateToProps: (state) => ({
       choices: (state.datasource) ? state.datasource.filterable_cols : [],
       datasource: state.datasource,
+      opChoices: ['in', 'not in'],
```

Comment:
I'm not sure if there's a good reason to have different `opChoices` for WHERE and HAVING, my feeling is that we could hard code a superset :) of both arrays in the Filter element itself.